dashboard-for-web/articles/web-dashboard-designer-mode/designing-dashboard-items/scatter-chart/conditional-formatting.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Conditional Formatting
+author: Margarita Zakhodyaeva
+---
+# Conditional Formatting
+
+Use conditional formatting to highlight points in a Scatter Chart dashboard item.
+
+![web-cf-scatter-chart-main](../../../../images/web-cf-scatter-chart-main.png)
+
+## Supported Format Rules
+
+You can use the following data in rule calculations:
+
+- [measures](../../binding-dashboard-items-to-data/binding-dashboard-items-to-data-in-the-web-dashboard.md) from the **X and Y axis** sections 
+- [measures](../../binding-dashboard-items-to-data/binding-dashboard-items-to-data-in-the-web-dashboard.md) from the **Weight** section
+- [dimensions](../../binding-dashboard-items-to-data/binding-dashboard-items-to-data-in-the-web-dashboard.md) from the **Arguments** section 
+- hidden measures 
+
+Format conditions that can be applied to different data item types are as follows:
+* numeric 
+	* **Value** 
+	* **Top-Bottom**
+	* **Average**
+	* **Expression** 
+	* **Color Ranges**
+	* **Gradient Ranges**
+* string 
+	* **Value** (with the condition type set to _Equal To_, _Not Equal To_ or _Text that Contains_)
+	* **Expression**
+* date-time 
+	* **Value**
+	* A **Date Occurring** (for dimensions with a continuous date-time group interval)
```

Comment:
Should this be "`**A Date Occurring**`" (Just checking.)

Reply:
```suggestion
	* **A Date Occurring** (for dimensions with a continuous date-time group interval)
```